scripts/launch-pos-eigen-zeth-node.sh

```diff
@@ -0,0 +1,11 @@
+cd ..
```

Comment:
We should target the script directory first by `DIR="$( cd "$( dirname "$0" )" && pwd )"`, then we can run the script from any relative folder.  Do it for other scripts. 

BTW, we should delete the old script for lauching the node.

Reply:
done!